docker/pulsar-all/pom.xml

```diff
@@ -139,6 +135,7 @@
                 <goals>
                   <goal>build</goal>
                   <goal>tag</goal>
+                  <goal>push</goal>
```

Comment:
We don't push the image by the maven, so we need to remove this goal.

Reply:
+1

Reply:
I think this is the reason why we need the push goal:

>Buildx is enabled when there is a non-empty <platform> element inside the <buildx> configuration. Only the native platform is built and saved in the local image cache by the build goal. All specified platforms are built and pushed into the remote repository by the push goal. This behavior is to prevent non-native images from tainting the local image cache.